pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -4236,5 +4236,84 @@ public void setSchemaValidationEnforced(@Suspended AsyncResponse asyncResponse,
                 });
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/{topic}/shadowTopic")
```

Comment:
Should the name be 'plural' ? shadowTopics

Reply:
@eolivelli Make sense, addressed.